cmd/kube-proxy/app/server.go

```diff
@@ -699,3 +713,20 @@ func getNodeIP(client clientset.Interface, hostname string) net.IP {
 	}
 	return nodeIP
 }
+
+func isLocalIP(ip string) (bool, error) {
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		return false, err
+	}
+	for i := range addrs {
+		intf, _, err := net.ParseCIDR(addrs[i].String())
+		if err != nil {
+			return false, err
+		}
+		if net.ParseIP(ip).Equal(intf) {
```

Comment:
Better to parse IP and pass to this function, instead of parsing for each check?